README.md

```diff
@@ -1,17 +1,62 @@
 [//]: # "This README.md file is auto-generated, all changes to this file will be lost."
 [//]: # "To regenerate it, use `python -m synthtool`."
-<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
-
-# [flaky.dev: a flaky test identification service](https://github.com/google/flaky-service)
 
 [![release level](https://img.shields.io/badge/release%20level-beta-yellow.svg?style=flat)](https://cloud.google.com/terms/launch-stages)
 [![npm version](https://img.shields.io/npm/v/flaky-service.svg)](https://www.npmjs.org/package/flaky-service)
 [![codecov](https://img.shields.io/codecov/c/github/google/flaky-service/master.svg?style=flat)](https://codecov.io/gh/google/flaky-service)
 
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
+
+# [flaky.dev: a flaky test identification service](https://github.com/GoogleCloudPlatform/flaky-service)
+
+### Overview
+Flaky.dev is a powerful test tracking service that helps you manage [flaky tests](https://hackernoon.com/flaky-tests-a-war-that-never-ends-9aa32fdef359). Configure your Github Workflow to automatically send your test logs to [Flaky.dev](%60https://flaky-dashboard.web.app), and watch your repositories test suite health improve from `https://flaky-dashboard.web.app/org/{My organization}` 
+
+
+### Getting started
+
+Your organization and repository pages on [Flaky.dev](%60https://flaky-dashboard.web.app) will be initialized as soon as you configure your [Github Workflow](https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow) to post your testing logs to our server. This generally consists of two steps: modifying/creating a your workflow's testing script to save your test results to disk in TAP format, and calling our custom Github action which posts these results to our server.
+
+
+#### 1: Produce test reports in the TAP format.
+
+This can certainly be easily done in your [favorite framework](https://www.npmjs.com/search?q=TAP+reporter). The example action below assumes you save your testing logs to a file titled `flaky-tap-log.tap` in the default github workspace directly. If you are using Node.js and mocha, creating this log may look something like this:
+
+`c8 mocha --reporter=tap test/*.js | flaky-tap-log.tap`
```

Comment:
We would need the piping to `tee` here too, I might just share this exact line:

```
 "test": "c8 mocha --reporter=tap | tee test.tap | tap-spec",
```

And mention that the `tee` bin outputs to standard out and a file at the same time, and the dependency `tap-spec` is optional, and coverts tap output to something closer to mocha's spec output.